### PR TITLE
update default PHP version to 7.4

### LINKED ISSUE #1877: php 7.2 EOL'ed

### PR DESCRIPTION
**Pull request by kayiwa**
closes #1877

### CODE REVIEW

File: playbooks/pas_production.yml [hunk 1]
```diff
@@ -7,11 +7,8 @@
     - ../group_vars/pas/vault.yml
   vars:
     - pas_upload_path: 'pas/pas-production'
-  pre_tasks:
-    - set_fact:
-        deploy_id_rsa_private_key: "{{  lookup('file', '../roles/pas_code/files/id_rsa')  }}\n"
   roles:
-    - role: roles/datadog
+    #    - role: roles/datadog
```
Comment (carolyncole) [inline playbooks/pas_production.yml:11]: This needs to be put back in

File: roles/libwww/tasks/main.yml [hunk 2]
```diff
@@ -73,36 +73,36 @@
...
-- name: ensure max file size is set in apache2 php.ini
+- name: libwww | ensure max file size is set in apache2 php.ini
   lineinfile:
-    dest: "/etc/php/7.2/apache2/php.ini"
+    dest: "/etc/php/7.4/apache2/php.ini"
     regexp: "upload_max_filesize ="
     line: "upload_max_filesize = 8M"
 
-- name: ensure memory limit is set in apache2 php.ini
+- name: libwww | ensure memory limit is set in apache4 php.ini
```
Comment (carolyncole) [inline roles/libwww/tasks/main.yml:90]: Is this really apache4?

File: roles/pas/vars/main.yml [hunk 3]
```diff
@@ -5,6 +5,7 @@ capistrano_shared_links: []
 capistrano_base_dir: '/opt'
 capistrano_web_owner: 'www-data'
 capistrano_web_service: 'apache2'
+php_webserver: 'apache2'
```
Comment (carolyncole) [inline roles/pas/vars/main.yml:8]: Did we decide this was not needed

**Review by carolyncole — CHANGES_REQUESTED** [review]
A couple small changes...  Otherwise looks great.

Also who uses the matomo role.  I do not think we tested that.

File: roles/libwww/tasks/main.yml (hunk 2 above)
Comment (kayiwa) [reply roles/libwww/tasks/main.yml:90]: stupid perl scripts.

**Comment by kayiwa** [comment]
> A couple small changes... Otherwise looks great.
> 
> Also who uses the matomo role. I do not think we tested that.

I am pretty sure we are shutting that down this month. (I've said that for the last three months.)

**Review by carolyncole — APPROVED** [review]

**Pull request merged**